blocks/subscriptions-block/components/OrderInformation/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import { usePhrases, Link, Heading, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+import { FeatureDetails } from "../OfferCard";
+import currency from "../../utils/currency";
+
+const OrderSummary = ({ orderDetails, className }) => {
+	const phrases = usePhrases();
+
+	return (
+		<Stack className={`${className}__summary`}>
+			<Heading>{phrases.t("checkout-block.order-summary")}</Heading>
+			<Stack className={`${className}__summary--details`}>
+				<Stack direction="horizontal" justification="space-between" className={`${className}__summary--details--item`}>
+					<p>{phrases.t("checkout-block.subtotal")}</p>
+					<p>{`${currency(orderDetails?.currency)}${orderDetails?.subtotal}`}</p>
+				</Stack>
+				<Stack direction="horizontal"  justification="space-between" className={`${className}__summary--details--item`}>
+					<p>{phrases.t("checkout-block.salesTax")}</p>
+					<p>
+						{orderDetails?.tax > 0
+							? `${currency(orderDetails?.currency)}${orderDetails?.tax}`
+							: "--"}
+					</p>
+				</Stack>
+			</Stack>
+			<div className={`${className}__summary--dueToday`}>
+				<p>{phrases.t("checkout-block.due-today")}</p>
+				<p>{`${currency(orderDetails?.currency)}${orderDetails?.total}`}</p>
+			</div>
+		</Stack>
+	);
+};
+
+const ProductPriceDetails = ({
+	details = [],
+	showPriceDescription,
+	showProductFeatures,
+	className,
+}) => {
+	if (details?.items?.length) {
+		return details?.items?.map((item) => 
+				<Stack as="div" className={`${className}__orderCard--productPrice`}>
+					<Heading> {item.priceName}</Heading>
+					{showPriceDescription && (
+						<Paragraph dangerouslySetInnerHTML={{ __html: item?.priceDescription }} />
+					)}
+					{showProductFeatures && (
+						<FeatureDetails
+							features={item?.productAttributes}
+							className={`${className}__orderCard`}
+						/>
+					)}
+				</Stack>
+		);
+	}
+	return null;
+};
+
+const OrderInformation = ({
+	id,
+	offerURL,
+	showOfferURL,
+	showPriceDescription,
+	showProductFeatures,
+	orderDetails,
+	className,
```

Comment:
The ticket mentions passing 
b. paymentType: ‘stripeIntents’ || ‘paypal’ || undefined
    c. campaignName
    Do we no longer needs these passed to OrderInformation component?